bundler/lib/bundler/cli/outdated.rb

```diff
@@ -155,7 +155,7 @@ def retrieve_active_spec(definition, current_spec)
 
       return active_spec if strict
 
-      active_specs = active_spec.source.specs.search(current_spec.name).select {|spec| spec.match_platform(current_spec.platform) }.sort_by(&:version)
+      active_specs = active_spec.source.specs.search(current_spec.name).select {|spec| spec.installable_on_platform?(current_spec.platform) }.sort_by(&:version)
```

Comment:
I like the new name much better and had thought in the past of renaming it to something along those lines 👍. Just wondering if we should keep the previous name as a deprecated alias.

Reply:
I think we're probably OK not to?